roles/result/tasks/main.yml

```diff
@@ -0,0 +1,41 @@
+---
+
```

Comment:
We should also display the results, right?

Reply:
I'm against it, because the logs would be super ugly, if the user wants to look at results, they can always look at logs of the pods.

Reply:
How do you figure it would be ugly? If they look at the pod, it will be the same output :)

Reply:
Yes, exactly my point if they can already get the output by looking at the pod, why also push it to logs of the operator pod which is already quite difficult to read.